### ISSUE TITLE
Load (more) complete gene-gene network into the database

### ISSUE THREAD

**Issue by dongbohu** [greenelab/adage-server#183]
Currently the gene-gene network file that has been loaded into the database has a cutoff value of 0.4. This file includes *139,111* gene pairs. 

Ideally we should load all gene-gene pairs into the database without any cutoff value. According to @tj8901nm, the complete gene-gene network file includes 5,549 * 5,548 / 2 = 15,392,926 gene pairs. Although such a big file is likely to hurt the performance of both back end and front end, we should load it to get a better idea of how much it will affect the server, and possibly set a lower cutoff value with acceptable performance.

**Comment by dongbohu** [comment]
Location of the complete gene-gene network file:

```
Dropbox/GreeneLab/ADAGE_server/gene-gene_network_complete.txt
```

File size: 507 MB

**Comment by dongbohu** [comment]
Two genes `PA2245` and `PA4638` are not found in gene info, so the number of genes that are recognizable is 5,549 - 2 = **5,547**, and the number of gene-gene network records loaded into the database is: 5547 * 5546 / 2 = **15,381,831**.

**Comment by dongbohu** [comment]
Here is a brief summary of edges weights in the complete gene-gene network:
```
0.0 to 0.1:  8,683,750
0.1 to 0.2:  4,560,391
0.2 to 0.3:  1,579,674
0.3 to 0.4:    430,000
0.4 to 0.5:    101,042
0.5 to 0.6:     24,865
0.6 to 0.7:      7,728
0.7 to 0.8:      3,475
0.8 to 0.9:      1,675
0.9 to 1.0:        326
Total:      15,392,926
```
It shows that ~86% edges' weight are less than 0.2, which most users may not be interested in. After talking with @tj8901nm and @cgreene, we decided to upload only the edges whose weight is >= 0.2, instead of loading the complete gene-gene network.

**Comment by dongbohu** [comment]
Fixed in PR #255.
